### PR TITLE
[draft] Motion components

### PR DESCRIPTION
**Pull request by robertpenner**
### New Motion Components
- **Fade**
- **Collapse**
- **Scale**

These motion components are React tags that wrap around content and add transitions to it. They are **presence** components, which means they have enter and exit motions to transition the content's presence on screen.

### New Motion APIs
- createPresenceComponentVariant()

## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/Avatar-mount_base.html">644</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/Avatar-mount.html">641</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/Button-mount_base.html">291</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/Button-mount.html">310</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/Field-mount_base.html">1125</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/Field-mount.html">1127</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/FluentProvider-mount_base.html">705</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/FluentProvider-mount.html">707</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/FluentProviderWithTheme-mount_base.html">85</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/FluentProviderWithTheme-mount.html">82</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">34</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">37</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">78</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">81</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/MakeStyles-mount_base.html">849</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/MakeStyles-mount.html">872</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/Persona-mount_base.html">1767</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/Persona-mount.html">1749</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/SpinButton-mount_base.html">1370</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/SpinButton-mount.html">1435</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/SwatchPicker-mount_base.html">1635</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31705/perf-test-react-components/SwatchPicker-mount.html">1664</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: apps/public-docsite-v9/package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "@fluentui/react-icons-northstar": "*",
```
Comment (fabricteam) [inline apps/public-docsite-v9/package.json:33]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31705&commitId=4e944ae66179b748be1e615dc513498bf0ef0b07&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|5| Changed|
|Avatar Converged.basic - RTL.normal.chromium.png|0| Removed|
|Avatar Converged.size+inactive+badge.normal.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Comment by layershifter** [comment]
Closing for housekeeping 🏡  Feel free to re-open

**Pull request closed without merging**